script.js

```diff
@@ -0,0 +1,85 @@
+const form = document.getElementById('form');
+let books = JSON.parse(localStorage.getItem('books'));
```

Comment:
- [ ] Kindly move the books array to be a member inside the `Book` class as per this [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m2_plain_js_classes_v1_1.md#:~:text=You%20can%20use%20objects%20and%20arrays%2C%20but%20do%20it%20inside%20classes). You can define a getter method inside the class to get all books in order to use it outside the class.

Reply:
@gemmen29 I defined them globally it makes a problem when I move it to the class or to make a getter for.

Reply:
And that array is or getting an item from the local storage. needs to be outside.
@gemmen29

Reply:
Hi @eaesmat you can define a method to get all books inside the class and use this method instead of the global object.